### PR TITLE
maxwell branch prettified

### PR DESCRIPTION
**Pull request by pnavaro**

### CODE REVIEW

**Comment by pnavaro** [comment]
@kkormann I formatted the files with fprettify on both sides (main and maxwell) and now the conflicts are easier to fix

**Comment by pnavaro** [comment]
I fixed the conflicts but i get this error

~~~
/Users/navaro/Codes/selalib/simulations/parallel/pic_vm_1d2v_cart/sll_m_sim_pic_vm_1d2v_cart.F90:36:8:

   36 |    use sll_m_hamiltonian_splitting_pic_vm_1d2v, only: &
      |        1
~~~

**Comment by pnavaro** [comment]
I disabled the github action that format the files. It will be easier for you for future merge

**Comment by kkormann** [comment]
For me the branch compiles on my computer. I merged the main and see if this fixes the checks...

**Comment by pnavaro** [comment]
Dear Katharina you can see the differences between your version and mine and how i fix the failing test spline_pp_2d

https://github.com/selalib/selalib/commit/2e194c8a0654fbf6d6b33cb8fce5005ed578b8a8#diff-b6fc122d3c662a999ac21e894d8f2364fbee425a96788c1c48eed85a10d6f87e

I add all these  `if (fail) stop 'FAILED'` to find where the test is failing

File: src/splines/splines_basic/testing/test_splines_pp_2d.F90 [hunk 1]
```diff
@@ -146,7 +146,7 @@ subroutine spline_test(spline_pp, degree, n_cells, boundary, xnorm, fail)
       sll_int32, intent(in) :: n_cells(2) !grid cells
       sll_int32, intent(in) :: boundary(2) ! boundaries
       sll_real64, intent(in) :: xnorm(2)
-      logical, intent(out)    :: fail
+      logical, intent(inout)    :: fail
```
Comment (kkormann) [inline src/splines/splines_basic/testing/test_splines_pp_2d.F90:149]: This was the problem causing the tests to fail about every second time....

**Comment by kkormann** [comment]
@pnavaro Now all the tests are running. Do you have anything else that should be cleaned before we merge?

**Comment by pnavaro** [comment]
Great, you ca merge the PR. Thanks Katharina !

**Comment by kkormann** [comment]
I will merge now. Thanks, Pierre, for your help!

**Pull request merged**